crates/amalthea/src/comm/data_explorer_comm.rs

```diff
@@ -383,38 +383,38 @@ pub struct SummaryStatsString {
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct SummaryStatsDate {
 	/// The exact number of distinct values
-	pub num_unique: i64,
+	pub num_unique: Option<i64>,
 
 	/// Minimum date value as string
-	pub min_date: String,
+	pub min_date: Option<String>,
 
 	/// Average date value as string
-	pub mean_date: String,
+	pub mean_date: Option<String>,
```

Comment:
Are we really showing mean date? That doesn't mean anything practical

Reply:
We're not showing it in the UI, even though we compute it. I had the same feeling here: https://github.com/posit-dev/positron/pull/4345#pullrequestreview-2238371237